evm/its.js

```diff
@@ -57,6 +58,24 @@ async function handleTx(tx, chain, contract, action, firstEvent, secondEvent) {
     }
 }
 
+const decodeMulticallData = async (encodedData, contractJSON) => {
+    const decodedArray = defaultAbiCoder.decode(['bytes[]'], encodedData)[0];
+    const iface = new Interface(contractJSON.abi);
+
+    return decodedArray.map((encodedCall) => {
+        try {
+            const parsedCall = iface.parseTransaction({ data: encodedCall });
+            const functionName = parsedCall.name;
+            const args = parsedCall.args.map((arg) => arg.toString());
+
+            return ` ${functionName}(${args.join(', ')})`;
```

Comment:
```suggestion
            return `${functionName}(${args.join('\n')})`;
```
display on multiple lines maybe?

Reply:
I updated the format a little to make it more readable, adding an example to the readme, let me know what you think